app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -69,6 +69,20 @@ class Config {
     return String.fromCharCodes(cacheValue).split(',');
   }
 
+  Future<List<Map<String, dynamic>>> _getLuciBuilders(String bucket) async {
+    final Uint8List cacheValue = await _cache.getOrCreate(
```

Comment:
Should we bypass the cache? this will force people to wait up to 15 mins to see the change.

Reply:
SGTM. Updated.